x-pack/plugins/security_solution/cypress/integration/cases/creation.spec.ts

```diff
@@ -85,10 +83,8 @@ describe('Cases', () => {
     cy.get(ALL_CASES_CLOSED_CASES_STATS).should('have.text', '0');
     cy.get(ALL_CASES_IN_PROGRESS_CASES_STATS).should('have.text', '0');
     cy.get(ALL_CASES_OPEN_CASES_COUNT).should('have.text', 'Open (1)');
-    cy.get(ALL_CASES_REPORTERS_COUNT).should('have.text', 'Reporter1');
```

Comment:
The reporter column no longer exists in the cases list page. It is now the assignees column but there won't be any assignees by default.